cabra/src/pages/Home.tsx

```diff
@@ -0,0 +1,24 @@
+import tw, { styled } from "twin.macro";
+
+import RandomGoat from "./components/RandomGoat";
+import TaskForm from "./components/TaskForm";
+import TasksList from "./components/TasksList";
+
+const Container = styled.div`
+  ${tw`flex flex-col items-center h-screen py-10 overflow-auto max-h-screen`}
+  ${tw`bg-gradient-to-b from-blue-700 to-fuchsia-500`}
+`;
```

Comment:
Looks like this is duplicated across all pages so it'd be nice to extract it into some kind of `Layout` component and use that instead